src/components/news_story/index.tsx

```diff
@@ -1,33 +1,13 @@
 import { ComponentType } from 'react'
-import { ImageProps, MediaImageComponent } from '@/components/media'
-import {
-  PersonProfileTeaserProps,
-  StaffNewsProfile,
-} from '@/components/person_profile'
 import { formatDate, truncateWordsOrChar } from '@/utils/helpers'
-import { SocialLinks, SocialLinksProps } from '@/components/socialLinks'
```

Comment:
Reorganized these. More global/helpers first; the Prop types second; any components imported for use third.